lib/thumbor_rails/helpers.rb

```diff
@@ -8,7 +8,11 @@ def thumbor_url(image_url, options = {})
       if ThumborRails.force_no_protocol_in_source_url
         image_url = image_url.sub(/^http(s|):\/\//, '')
       end
-
+      
+      unless options[:unsafe]
+        image_url = ERB::Util.url_encode(image_url)
```

Comment:
It's for more comfortable using on browser code inspector (for change size or filter). Encoded symbols like & ? (%26%20%3F) poor for editing.